ledger/accountdb.go

```diff
@@ -303,54 +701,34 @@ func (a *compactAccountDeltas) len() int {
 	return len(a.deltas)
 }
 
-func (a *compactAccountDeltas) getByIdx(i int) (basics.Address, accountDelta) {
-	return a.addresses[i], a.deltas[i]
-}
-
-// upsert updates existing or inserts a new entry
-func (a *compactAccountDeltas) upsert(addr basics.Address, delta accountDelta) {
-	if idx, exist := a.cache[addr]; exist { // nil map lookup is OK
-		a.deltas[idx] = delta
-		return
-	}
-	a.insert(addr, delta)
+func (a *compactAccountDeltas) getByIdx(i int) accountDelta {
+	return a.deltas[i]
```

Comment:
Now that this function (and `compactResourceDeltas.getByIdx`) are so simple, I think calling code would be clearer if it just said `a.deltas[i]` and these functions removed.  By using this function, which doesn't have the word "delta" in it, the meaning is obscured.